css/css-device-adapt/META.yml

```diff
@@ -0,0 +1,4 @@
+spec: https://drafts.csswg.org/css-device-adapt/
+suggested_reviewers:
+  - ChumpChief
+  - frivoal
```

Comment:
@frivoal I see you are listed as editor of CSS Device Adaptation, would you like to get reviews for this directory?